ci/kubetest/test_clickhouse_persistence_disabled.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from utils import cleanup_k8s, get_clickhouse_statefulset_spec, helm_install, wait_for_pods_to_be_ready
+
+HELM_INSTALL_CMD = """
+helm upgrade \
+    --install \
+    -f ../../ci/values/kubetest/test_clickhouse_persistence_disabled.yaml \
+    --timeout 30m \
+    --create-namespace \
+    --namespace posthog \
```

Comment:
non-blocking

Worth putting these test cases into separate namespaces for better isolation?

Reply:
It could be nice 😅 the problem is that I don't think our chart works in a namespace != posthog (see [here](https://github.com/PostHog/charts-clickhouse/issues/120))

Reply:
Ah ok cool, in my test I did use non-posthog namespace but yeah as long as the tests are passing and checking the right things I don't think it matters.